apps/web/lib/event-types/getServerSideProps.tsx

```diff
@@ -0,0 +1,23 @@
+import type { GetServerSidePropsContext } from "next";
+
+import { getServerSession } from "@calcom/features/auth/lib/getServerSession";
+
+import { ssrInit } from "@server/lib/ssr";
+
+export const getServerSideProps = async (context: GetServerSidePropsContext) => {
+  const ssr = await ssrInit(context);
+  const session = await getServerSession({ req: context.req, res: context.res });
+
+  if (!session) {
+    return {
+      redirect: {
+        redirect: {
+          permanent: false,
+          destination: "/auth/login",
+        },
+      },
+    };
+  }
+
+  return { props: { trpcState: ssr.dehydrate() } };
+};
```

Comment:
do we even need this if we remove the feature flag for managed event types?

Reply:
i think we should still include this.

Reply:
ok.

btw. end to end tests failing, not sure if its related to this PR @keithwillcode ?

Reply:
It’s not unfortunately. We have E2E test suite issues at the moment. Hoping to resolve by tomorrow and will get this merged.